development/wasm-filter-plugins.md

```diff
@@ -128,6 +128,38 @@ For example, one of the examples of [Rust WASM filter](https://github.com/fluent
 [0] dummy.local: [1666270589.270348000, {"lang"=>"Rust", "message"=>"dummy", "original"=>"{"message":"dummy"}", "tag"=>"dummy.local", "time"=>"2022-10-20T12:56:29.270348000 +0000"}]
 [0] dummy.local: [1666270590.271107000, {"lang"=>"Rust", "message"=>"dummy", "original"=>"{"message":"dummy"}", "tag"=>"dummy.local", "time"=>"2022-10-20T12:56:30.271107000 +0000"}]
 ```
+Another example of a Rust WASM filter is the [flb_filter_iis](https://github.com/kenriortega/flb_filter_iis) filter.
+This filter takes the [Internet Information Services (IIS)](https://learn.microsoft.com/en-us/iis/manage/provisioning-and-managing-iis/configure-logging-in-iis) [w3c logs](https://learn.microsoft.com/en-us/iis/manage/provisioning-and-managing-iis/configure-logging-in-iis#select-w3c-fields-to-log) (with some custom modifications) and transforms the raw string into a fluentbit json standard.
+
+```text
+[INPUT]
+    Name             dummy
+    Dummy            {"log": "2023-08-11 19:56:44 W3SVC1 WIN-PC1 ::1 GET / - 80 ::1 Mozilla/5.0+(Windows+NT+10.0;+Win64;+x64)+AppleWebKit/537.36+(KHTML,+like+Gecko)+Chrome/115.0.0.0+Safari/537.36+Edg/115.0.1901.200 - - localhost 304 142 756 1078 -"}
+    Tag              iis.*
+
+[FILTER]
+    Name             wasm
+    match            iis.*
+    WASM_Path        /plugins/flb_filter_iis_wasm.wasm
```

Comment:
Can your repo provide the WASM output directly as a release artefact (i.e. download from a specific release page)? I'm not sure what is possible with WASM but it would be nice to have a simple way to just insert the binary without having to compile everything.

Reply:
Ok, I will be working on it

Reply:
Maybe add a comment somewhere about where to get this from? I see you have the release now so a simple curl to grab it.